string mission
string behavior
uint32 index
string localizer
string mission_pln
string behavior_pln
string motion_pln
string lat_con
string lon_con
